### PR TITLE
feat: add requestid middleware examples

### PR DESCRIPTION
**Pull request by justlorain**
#### What type of PR is this?

feat: A new feature

#### What this PR does / why we need it (English/Chinese):

en: Add requestid middlerware examples
zh: 添加requestid中间件的使用示例

#### Which issue(s) this PR fixes:

Fixes (https://github.com/hertz-contrib/requestid/issues/6)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,13 +7,14 @@ require (
 	github.com/alibaba/sentinel-golang v1.0.4
 	github.com/aliyun/alibaba-cloud-sdk-go v1.61.1764 // indirect
 	github.com/apache/thrift v0.13.0
-	github.com/cloudwego/hertz v0.3.2
+	github.com/cloudwego/hertz v0.4.0
```
Comment (li-jin-gou) [inline go.mod:10]: use `go mod tidy && go mod verify`

Comment (justlorain) [reply go.mod:10]: done, thx

File: go.mod [hunk 2]
```diff
@@ -30,15 +32,14 @@ require (
 	github.com/kitex-contrib/tracer-opentracing v0.0.3
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/prometheus/client_golang v1.13.0 // indirect
-	github.com/sirupsen/logrus v1.9.0 // indirect
+	github.com/sirupsen/logrus v1.9.0
```
Comment (li-jin-gou) [inline go.mod:35]: add some licenses in licenses's dir if  it is direct dependency.

Comment (justlorain) [reply go.mod:35]: done, thx

**Review by li-jin-gou — APPROVED** [review]

File: middleware/requestid/default/main.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
+/*
```
Comment (li-jin-gou) [inline middleware/requestid/default/main.go:1]: I don't think this is necessary.

**Pull request merged**